build/test-integration.sh

```diff
@@ -9,12 +9,13 @@ set -e
 declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
 source $REPO_DIR/build/__variables.sh
 
-if [ -z "$STORAGEACCOUNTKEY" ]; then # If STORAGEACCOUNTKEY is unset or empty
-	export STORAGEACCOUNTKEY=`az.cmd storage account keys list -n oryxautomation | grep key1 | awk '{print $NF}'`
-fi
+# if [ -z "$STORAGEACCOUNTKEY" ]; then # If STORAGEACCOUNTKEY is unset or empty
```

Comment:
Did you want it to be removed?

Reply:
No - but it's not needed as long as we're temporarily skipping AKS tests

Reply:
Yeah, we still have these artifacts from AKS. We could probably do a separate issue-PR for completely removing these artifacts, but yeah not a high pri thing.